features/26-mfa/api.md

```diff
@@ -0,0 +1,436 @@
+# Overview
+
+The following factors are supported:
+
+- Time-Based One-Time Password [(TOTP)](https://tools.ietf.org/html/rfc6238)
+- Out-of-band
+  - SMS
```

Comment:
I intentionally left it out because access to an email is not an additional factor. Access to an email is "something they know" which is the same factor as password.

Reply:
It depends, if the users use loginWithSMS / forgotPasswordWithSMS something like that, than SMS is not another factor lol. But yea this is not very important... just nice to have if not complicated.

Reply:
OK. Let me add it as a channel of Out-of-band.